INCLUDED-MODS.md

```diff
@@ -0,0 +1,100 @@
+## Included mods
```

Comment:
oh yeah, this needs to be # and ### needs to be ##

Reply:
Wdym, there's no `###` in the readme, and the only `#` is the title

Reply:
the point is that sub-sub-headlines become sub-headlines and sub-headlines become headlines

Reply:
I thought it was the readme file